spec/nest_thermostat_spec.rb

```diff
@@ -1,9 +1,13 @@
 require 'spec_helper'
 
 module NestThermostat
-  describe Nest do
+  # Run all the requests through VCR, but don't match on the host or URI as they vary
+  # match on all the other parts of the request to find the unique request in the cassette
+  describe Nest, :vcr => { :match_requests_on => [:method, :path, :query, :body]} do
     before(:all) do
-      @nest = Nest.new(email: ENV['NEST_EMAIL'], password: ENV['NEST_PASS'], temperature_scale: :fahrenheit)
+      VCR.use_cassette("connect to api") do
+        @nest = Nest.new(email: 'test@yahoo.com' , password: 'sekret', temperature_scale: :fahrenheit)
```

Comment:
Line is too long. [102/80]<br>Space found before comma.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.